src/App.css

```diff
@@ -1,11 +1,12 @@
-@import url('https://fonts.googleapis.com/css2?family=Roboto');
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@300;400;700&display=swap');
```

Comment:
Your project imports and uses the Montserrat font family, but not the RobotoSlab which is required in several elements:

![image](https://user-images.githubusercontent.com/57421823/161025887-1a688afd-3599-4e03-851a-e58e2e0c7452.png)

Therefore:

- [ ] Please, import and implement the RobotoSlab font family where it's required, according to the Zeplin.  Please remember to check that the exact font styling matches each element.

That way, your project will look like the Zeplin, and we will meet this requirement. 💪

Reply:
Thanks for catching that!